mmds/src/data_store.rs

```diff
@@ -117,18 +147,24 @@ mod tests {
     #[test]
     fn test_mmds() {
         let mut mmds = Mmds::default();
-        assert_eq!(mmds.is_initialized(), false);
+
+        assert_eq! {
```

Comment:
Aren't these supposed to be parantheses?

Reply:
Wooooow, this compiles?

Reply:
Macro invocations in Rust accept either `(`, `{`, or `[`, as long as you close the parameter list with a matching symbol :D That's why you can use `vec!` for example as `let something = vec![1, 2, 3]`.

Reply:
Still, all the other `assert!`s use parantheses.

Reply:
Sorry for this :(